cmd/node/main.go

```diff
@@ -1192,6 +1202,28 @@ func createNodesCoordinator(
 		return nil, err
 	}
 
+	thresholdEpochDuration := 0.25
+	maxDurationBeforeStopProcess := int64(nodesConfig.RoundDuration) * roundsPerEpoch
+	maxDurationBeforeStopProcess = int64(thresholdEpochDuration * float64(maxDurationBeforeStopProcess))
+	intRandomizer := &random.ConcurrentSafeIntRandomizer{}
+	randDurationBeforeStop, err := intRandomizer.Intn(int(maxDurationBeforeStopProcess))
+	if err != nil {
+		return nil, err
+	}
+	endOfProcessingHandler := func() error {
+		go func() {
```

Comment:
no need for go func

Reply:
it is better this way. Otherwise, imagine when you start the node in a wrong shard. the channel will be written in it. but the checks for that channel are done only at the end of startNode() method in main. Also the print would be placed in a wrong place. 
Also, if I wouldn't use a goroutine, the function would be blocking. So what need for a random wait time before node stop ? if the node doesn't do anything meanwhile.